images/pulsar-functions-base-runner/Dockerfile

```diff
@@ -53,6 +53,12 @@ RUN echo "VERSION_TAG=$(VERSION_TAG)" && \
          && apt-get clean \
          && rm -rf /var/lib/apt/lists/*
 
+ADD https://github.com/fullstorydev/grpcurl/releases/download/v1.8.7/grpcurl_1.8.7_linux_x86_64.tar.gz /pulsar/grpcurl.tar.gz
+RUN tar -xvf /pulsar/grpcurl.tar.gz -C /pulsar/bin/ && \
+    rm -rf /pulsar/grpcurl.tar.gz
+COPY InstanceCommunication.proto /pulsar/conf/InstanceCommunication.proto
```

Comment:
is it possible to use the proto file from pulsar repo?

Reply:
yes